### DISCUSSION TITLE
command completion notification disable

### DISCUSSION THREAD

**Discussion by semi710** [marlonrichert/zsh-snap#263] [General]
every time a command get's completed i get a notification how can i disable it?

**Comment by marlonrichert** [comment]
Can you edit your question and add a screenshot or something like that? I have no idea what you're talking about.

**Reply by semi710** [reply]
![image](https://github.com/marlonrichert/zsh-snap/assets/60490474/29c169d9-cf57-4bdd-b85a-889c3a79b975)
whenever a command takes longer and get's completed it sends a notification after the command completion.

not 100% sure if from znap but tried disabling from zsh and i face it still
```zsh
unset REPORTTIME
unsetopt NOTIFY # disables comman finish notifications
```

Tried these do no luck

**Reply by marlonrichert** [reply]
Not from Znap and not from Zsh. Might be from your terminal.
